packages/caliper-fabric/test/FabricConnectorFactory.js

```diff
@@ -207,15 +171,13 @@ describe('A Fabric Connector Factory', () => {
     it('should throw an error if fabric library bound is not V1 or V2', async () => {
         mockery.registerMock('fabric-network', {});
         mockery.registerMock('fabric-network/package', {version: '3.0.0'});
-        mockery.registerMock('fabric-client', {});
-        mockery.registerMock('fabric-client/package', {version: '3.0.0'});
         ConfigUtil.set(ConfigUtil.keys.NetworkConfig, path.resolve(__dirname, v2Config));
         ConfigUtil.set(ConfigUtil.keys.Fabric.Gateway.Enabled, true);
         await ConnectorFactory(1).should.be.rejectedWith(/Installed SDK version 3.0.0 did not match any compatible Fabric connectors/);
         mockery.deregisterAll();
     });
 
-    it('should throw an error version if not for legacy or the new connector', async () => {
+    it('should throw a generic error if network configuration version is1.0 or 2.0', async () => {
```

Comment:
The description is a bit unclear. Shouldn't it end with `version is not 2.0`

Reply:
Well spotted. I blame my keyboard, it's wireless and so may times it doesn't register keystrokes but it's ergonomic :-)